main/cmds.go

```diff
@@ -40,11 +40,11 @@ var (
 
 	// allowed user inputs
 	cmds = map[string]cmd{
-		"install":   {install, "install", false, nil, 52},
-		"uninstall": {uninstall, "uninstall", false, nil, 3},
-		"enable":    {enable, "enable", true, enablePre, 3},
-		"update":    {update, "update", true, nil, 3},
-		"disable":   {disable, "disable", true, nil, 3},
+		"install":   {install, "install", false, nil, 100},
+		"uninstall": {uninstall, "uninstall", false, nil, 200},
+		"enable":    {enable, "enable", true, enablePre, 300},
+		"update":    {update, "update", true, nil, 400},
+		"disable":   {disable, "disable", true, nil, 500},
```

Comment:
Please add the codes in the documentation.

Reply:
Also update the errors codes here - https://github.com/Azure/Guest-Configuration-Extension/blob/megup-errorCodes/main/main.go#L113

I'll recommend creating a new ErrorCodes.go which only contain the error codes and mapping. And then use it internally everywhere.